conference/cfp/models.py

```diff
@@ -5,7 +5,7 @@
 from django.utils.translation import gettext_lazy as _
 
 
-class Submission(models.Model):
+class AbstractSubmission(models.Model):
```

Comment:
For compatibility, please add `@python_2_unicode_compatible` (see [the docs](https://docs.djangoproject.com/en/1.10/ref/utils/#django.utils.encoding.python_2_unicode_compatible)).